pycozmo/activity.py

```diff
@@ -32,7 +36,10 @@ def __init__(self,
 
         self.choice_type = str(choice_type)
         self.behaviors = behaviors
-        self.iteration = 0
+
+        if self.choice_type == 'Scoring':
+            self.init_scores()
+            self.init_repetition_penalty()
```

Comment:
Is repetition penalty specific to the scoring behavior chooser?

Reply:
Yes, it is (as long as I have seen). There is only one json with 'Selection' type, which does not contain any other info (actvitities_config.json), and the 'Strict' types always return the same behavior progression.